src/guide/1/essentials/security/index.md

```diff
@@ -136,56 +142,58 @@ When applying a role to a profile, the role can be applied to all indexes and co
 For example, if we have a "publisher" role which allows any action on the `document` controller:
 
 ```js
-const publisherRole = {
-  controllers: {
-    document: {
-      actions: {
-        '*': true
+{
+  "controllers": {
+    "document": {
+      "actions": {
+        "*": true
       }
     }
   }
-};
+}
 ```
 
 Then we can declare three different profiles using this same role, each with varying levels of access based on the index and collection:
 
+* Applies the publisher role to all indexes and collections
+
 ```js
-const profile1 = {
-  policies: [
-    {roleId: 'publisherRole'}
+{
+  "policies": [
+    {"roleId": "publisherRole"}
   ]
-};
+}
+```
 
-const profile2 = {
-  policies: [
+* Applies the publisher role only to the index "index1" and all its collections
+
+```js
+{
+  "policies": [
     {
-      roleId: 'publisherRole',
-      restrictedTo: [{index: 'index1'}]
+      "roleId": "publisherRole",
+      "restrictedTo": [{"index": "index1"}]
     }
   ]
-};
+}
+````
+
+* Applies the publisher role only to the collections "foo" and "bar" in the index "index1", and then to the index "index2" and all its collections
 
-const profile3 = {
-  policies: [
+```js
+{
+  "policies": [
     {
-      roleId: 'publisherRole',
-      restrictedTo: [
-        {index: 'index1', collections: ['foo', 'bar']},
-        {index: 'index2'}
+      "roleId": "publisherRole",
+      "restrictedTo": [
+        {"index": "index1", "collections": ["foo", "bar"]},
+        {"index": "index2"}
       ]
     }
   ]
-};
+}
 ```
 
-These three profiles will provide the following restrictions:
-
-* users with `profile1` are allowed to use all `document` controller actions on all indexes and collections.
-* users with `profile2` are only allowed to use `document` controller actions on collections stored in index `index1`.
-* users with `profile3` are only allowed to use `document` controller actions on:
-  * all collections stored in index `index2`
-  * collections `foo` and `bar` stored in index `index1`.
-
 ---
 
 ## Writing complex permission rules
```

Comment:
About the last (unchanged) sentence `There are multiple ways of adding a business logic layer on top of the standard Kuzzle security one:`, I would reorder the items, and put:
1. write a pipe plugin 
2. use document validators
3. use closures (deprecated)

Reply:
Makes more sense that way :+1: